Comp2024/src/main/java/frc/robot/subsystems/HID.java

```diff
@@ -70,7 +84,7 @@ public void simulationPeriodic( )
    */
   private void initDashboard( )
   {
-    // Initialize dashboard widgets

```

Comment:
Please leave this comment in as well (for the next person that has less experience than you).

Reply:
fixed